com.woltlab.wcf/templates/trophyList.tpl

```diff
@@ -3,7 +3,7 @@
 {capture assign='contentHeader'}
 	<header class="contentHeader messageGroupContentHeader">
 		<div class="contentHeaderTitle">
-			<h1 class="contentTitle">{$category->getTitle()}</h1>
+			<h1 class="contentTitle">{$category->getTitle()} <span class="badge badgeInverse">{#$items}</span></h1>
```

Comment:
The badge is shown regardless if there are any items or not, please wrap the span inside `{if $items}…{/if}`. This applies to  other templates too, but is not required for those that will always contain items (such as users, groups or packages).